strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/PathPrivileges.java

```diff
@@ -1,23 +1,20 @@
 package org.carlspring.strongbox.users.domain;
 
-import org.carlspring.strongbox.authorization.dto.PrivilegeDto;
-import org.carlspring.strongbox.users.dto.UserPathPrivelegiesReadContract;
-import org.carlspring.strongbox.users.dto.UserPathPrivilegesDto;
-
-import javax.annotation.concurrent.Immutable;
 import java.io.Serializable;
 import java.util.Collections;
 import java.util.Set;
 
-import com.google.common.collect.ImmutableSet;
-import static java.util.stream.Collectors.toSet;
+import javax.annotation.concurrent.Immutable;
+
+import org.carlspring.strongbox.users.dto.PathPrivelegiesData;
+import org.carlspring.strongbox.users.dto.PathPrivilegesDto;
 
 /**
  * @author Przemyslaw Fusik
  */
 @Immutable
-public class AccessModelPathPrivileges
-        implements Serializable, UserPathPrivelegiesReadContract
+public class PathPrivileges
+        implements Serializable, PathPrivelegiesData
```

Comment:
There is a typo in `PathPrivelegiesData`. It should be `PathPrivilegesData`.

Reply:
fixed